scripts/send_readme_to_admin.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
```

Comment:
I think that file name don't need to to be `.sh`, only `send_readme_to_admin`

Reply:
I have changed the name send_readme_to_admin